spec/requests/posts_

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Kindly rename your file, in hurry, I think you forgot to add posts_spec.rb and wrote only posts_ because of which only tests for users are running and not posts. Fix it. 😄